src/chatlist.rs

```diff
@@ -172,8 +172,11 @@ impl Chatlist {
                 )
                 .await?
         } else if let Some(query) = query {
-            let query = query.trim().to_string();
-            ensure!(!query.is_empty(), "missing query");
+            let mut query = query.trim().to_string();
+            let regex = regex::Regex::new(r"\bunread\b").unwrap();
+            let only_unread = regex.find(&query).is_some();
```

Comment:
I'd not allow it to be anywhere. Better in some fixed place not to confuse a user trying to build some complex logic on this